.github/workflows/release.yml

```diff
@@ -531,6 +531,21 @@ jobs:
         sleep 3
         docker logs kong
 
+    - name: Smoke Tests - Version Test
+      run: |
+        workflow_version="$(
+          echo '${{ steps.metadata.outputs.kong-version }}' \
+            | sed -e 's@\.@\\\.@g'
+        )"
+
+        # confirm workflow's version and built container version match with
+        # dots escaped, and end-line delimited
+        if ! docker exec kong kong version | grep -E "${workflow_version}$"; then
```

Comment:
@curiositycasualty  will this match the exact kong version string such as "Kong Enterprise 3.2.2.0" or only the version number?

Reply:
This will match ONLY the trailing version number.
But the `$` at the back of the `grep` command means that it will match agains the end of the output from `kong version`.
E.g.:
  - "Kong Enterprise 3.2.2.0" **will** match
  - "King Kong 3.2.2.0" **will** match
  - "3.2.2.0" **will** match *but*
  - "3.2.2.0-dev" or "Kong Enterprise 3.2.2.0-dev" or "King Kong 3.2.2.0-monkey" will **NOT** match

Reply:
Thanks, Isa, good to know.